net/amule/Portfile

```diff
@@ -62,3 +63,7 @@ configure.cxxflags-append \
                     "-framework CoreFoundation -framework CoreServices -framework IOKit"
 configure.ldflags-append \
                     "-framework CoreFoundation -framework CoreServices -framework IOKit"
+
+post-activate {
+    system "${prefix}/bin/update-desktop-database ${prefix}/share/applications"
+}
```

Comment:
should this also be run when the port is uninstalled?

Reply:
Hmm, honestly I do not know. I think other ports do not do that, but it may be wrong. What do you think? If yes, then how?

Reply:
@reneeotten I am looking at existing ports now, and none seem to use `post-deactivate` closure for `desktop-file-utils`.

Reply:
@reneeotten Maybe merge?